rubicon_ml/client/mixin.py

```diff
@@ -282,6 +283,49 @@ def log_h2o_model(
 
         return artifact
 
+    @failsafe
+    def log_xgboost_model(
+        self,
+        xgboost_model: "xgb.Booster",
+        artifact_name: Optional[str] = None,
+        **log_artifact_kwargs: Any,
+    ) -> Artifact:
+        """Log an XGBoost model as a JSON file to this client object.
+
+        Please note that we do not currently support logging directly from the SKLearn interface.
```

Comment:
what do you mean by this?

Reply:
If users use the `XGBRegressor` or `XGBClassifier` classes that implement sklearn interface for xgboost this will not work. They would have to use `model.get_booster()` first and log that. An obvious addition is to check for that ourselves and then call `model.get_booster()` if relevant. 

I can add that in this PR if you want.

Reply:
ahh gotcha, nah this is fine for now